frontend/src/components/DropZone.js

```diff
@@ -37,6 +40,24 @@ const activeStyle = {
   borderColor: '#0c9bef',
 }
 
+/**
+ * Custom validator to reject the `.git` folder.
+ * @param {FileWithPath} file
+ * @returns {FileError|null}
+ */
+const fileCustomValidator = file => {
+  const isFileUnderGitDir =
+    file?.path.includes('/.git/') || file?.webkitRelativePath.includes('/.git/')
+
+  if (isFileUnderGitDir) {
+    return {
+      code: ErrorCode.FileInvalidType,
+      message: "Uploading git folder isn't supported",
```

Comment:
I would prefer to just ignore these files.

Reply:
Now it ignores it and uploads the rest but it shows the message shown below, do you mean ignoring it silently?

![image](https://user-images.githubusercontent.com/37152329/151057313-caf4ce2a-dd9b-4f97-b35c-03935c9417d1.png)

Reply:
It has been deployed to https://abdo-dev.staging.kitspace.dev/

Reply:
That warning disappears so quickly it seems useless anyway. I think it's fine to just ignore these files silently.

Reply:
Resolved in 56f9c4f.